### ISSUE TITLE
Cant see any changes that were inplemented from 1.6.1 to 1.6.4

### ISSUE THREAD

**Issue by ZeTBCN** [BrettRyland/BDArmory#549]
After downloading and redownloading the latest bdarmory release from both GitHub and spacedock I am stuck at version 1.6.4, even though it says I am up to date.

**Comment by BrettRyland** [comment]
v1.6.4.0 is the current version. If you're not seeing the recent changes listed in the changelog despite installing the latest version, then you likely have multiple BDArmory DLLs installed, however, without the KSP.log file, I can't analyse the issue.

**Comment by ZeTBCN** [comment]
Where can i find ksp log file?

**Comment by BrettRyland** [comment]
It's in the same folder as the KSP executable. If you're using steam, then right click on KSP -> Manage -> "Browse local files" should open up the correct folder.

**Comment by ZeTBCN** [comment]
[KSP.zip](https://github.com/BrettRyland/BDArmory/files/12138867/KSP.zip)

**Comment by BrettRyland** [comment]
OK, I'm not entirely certain, but it looks like the errors stem from the KSPBurst mod.
```
[ERR 14:38:47.312] [KSPBurst]: Burst compiler had errors

[ERR 14:38:47.312] [KSPBurst]: Burst package not found: Could not find burst package archive in C:\Program Files (x86)\Steam\steamapps\common\Kerbal Space Program\GameData\000_KSPBurst or directory in C:\Program Files (x86)\Steam\steamapps\common\Kerbal Space Program\PluginData
```
which is followed by a number of errors from B9PartSwitch (this may or not be directly related to the above).

Further on in the log, it's showing some errors in BDArmory when setting up some weapons from other mods, which may also be caused from the above or they could be due to the other mods simply not being compatible with BDA+ (e.g., NKDR is not compatible, but some users on the forums (e.g., Rakete) have managed to edit their configs to get it to work).

I also note that you're missing the Firespitter dependency of AirplanePlus, which is causing some errors too.

**Comment by ZeTBCN** [comment]
Ok thanks, i will try to fix most of them.

**Comment by ZeTBCN** [comment]
I think i fixed the Airplane+ one and also bdarmory weapons(well not fix but remove)
But i have no idea how to fix the main one

**Comment by ZeTBCN** [comment]
> I think i fixed the Airplane+ one and also bdarmory weapons(well not fix but remove) But i have no idea how to fix the main one

nvm now got more error because of firespitter

**Comment by ZeTBCN** [comment]
I deleated kspburst and installed bdarmory 1.6.4.1 but the stuff still dosent work

**Comment by BrettRyland** [comment]
I'd recommend using CKAN to manage your mods (at least, the ones that are in CKAN) to make sure you have all the dependencies for them.
You have quite a large number of mods, so something else you could do is to make a copy of KSP (it runs fine without steam) with a limited set of mods and get it working correctly with BDArmory first, then add in other mods until something breaks again to see where the problem lies.

**Comment by ZeTBCN** [comment]
I am already using ckan and now can I make a copy?

**Comment by BrettRyland** [comment]
Just copy the folder to a different location and run it by starting the KSP executable instead of through steam. KSP is quite nice in that respect.

**Comment by ZeTBCN** [comment]
So all I need to copy from the gamedata is 2 default copies and bdarmory? Also can CKan manage 2 game instances so I can eddit one without editing the other?(with Ckan)

**Comment by BrettRyland** [comment]
Yes, CKAN can manage multiple KSP installs. Just copy the entire KSP folder, then remove the extra mods from the GameData of the copied folder (either manually or via CKAN).

**Comment by ZeTBCN** [comment]
[KSP.zip](https://github.com/BrettRyland/BDArmory/files/12171255/KSP.zip)

**Comment by BrettRyland** [comment]
ModuleManager and PhysicsRangeExtender are dependencies of BDArmory, so you'll need to include those. CKAN should have installed them as dependencies.
Here is a ckan modpack with the mods I use in Runway Project (in Scott Manley's discord) [RWP_mods.ckan](https://github.com/BrettRyland/BDArmory/files/12171522/RWP_mods.ckan.txt), for which everything should be working correctly (you can install it via CKAN's "Install from .ckan" option).

Also, can you be more specific as to what you're expecting to be working but isn't?

**Comment by ZeTBCN** [comment]
Sorry forgot about those

**Comment by ZeTBCN** [comment]
I am trying to get new thins like terminal guidance and missile loft to appear but they dont

**Comment by BrettRyland** [comment]
OK, well BillNye added a bunch of stuff to the wiki on using loft guidance https://github.com/BrettRyland/BDArmory/wiki/1.2.5-Missile-configuration#3-loft-guidance-missiles.

I'm not too familiar with the missiles aspect of BDA, that's more the province of BillNye, Josue and SuicidalInsanity, however, setting `homingType = aamloft` in one of the missile configs brings up the appropriate lofting sliders in the part window:
![Screenshot_20230726_181413](https://github.com/BrettRyland/BDArmory/assets/18440126/1e421451-313d-42b3-b74c-e5a513bca333)
so it looks to be working correctly for me at least.

**Comment by ZeTBCN** [comment]
It goes ATGM/Cruise/Ballistic/Proportional/Argumented/AAM and then back to atgm

**Comment by BrettRyland** [comment]
It's a setting in config file of regular missiles. It doesn't look like it's selectable for MMGs currently. I'll have to check with BillNye about whether it's supposed to be.

**Comment by ZeTBCN** [comment]
Ohhh, also what about terminal guidence?

**Comment by ZeTBCN** [comment]
Ok, so where i can find the loft and terminal guidence settings? Just in the aim-120 and in the cruise missiles? I cant find them there

**Comment by BrettRyland** [comment]
You need to set `homingType = aamloft` in the missile config you want to convert (make a custom copy of the config instead of adjusting the original) to using lofting as I demonstrated in the picture above. Similarly with `terminalHomingType` if using lofting for terminal guidance.
BillNye explained all this in detail on the wiki https://github.com/BrettRyland/BDArmory/wiki/1.2.5-Missile-configuration.

**Comment by ZeTBCN** [comment]
Alr tnx

**Comment by ZeTBCN** [comment]
I have loft working but not the terminal guidence.
